is_core/generic_views/table_views.py

```diff
@@ -137,24 +137,35 @@ def _get_header_order_by(self, model, full_field_name):
     def _get_field_labels(self):
         return self.field_labels
 
+    def _get_resource_label(self, model, field_name):
```

Comment:
labely pro hlavičky v taulce se mají získávat z resource. Tady to bylo trošku zprasené. Brousím si zuby na to tohle kompletně nějak přepsat.